terraform/environments/maat/api-dns.tf

```diff
@@ -0,0 +1,15 @@
+######################################
+# DNS Record for Load Balancer
+######################################
+
+resource "aws_route53_record" "maat_api_lb_a_record" {
+  zone_id = data.aws_route53_zone.external.zone_id
+  name    = "${local.application_name}-cd-api.${data.aws_route53_zone.external.name}"
+  type    = "CNAME"
+  ttl     = "60"
```

Comment:
We can consider using an Alias record instead - https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/resource-record-sets-choosing-alias-non-alias.html